src/core/transformScriptSetup.ts

```diff
@@ -1,6 +1,7 @@
 import { capitalize } from '@vue/shared'
 import type { Node, ObjectExpression, Statement } from '@babel/types'
 import { notNullish, partition, uniq } from '@antfu/utils'
+import htmlTags from 'html-tags'
```

Comment:
try using `"@vue/compiler-dom`'s `parserOptions.isNativeTag`

Reply:
> try using `"@vue/compiler-dom`'s `parserOptions.isNativeTag`

That's a good idea. I've finished the unit tests~